src/modules/UI/scenes/SendConfirmation/reducer.js

```diff
@@ -72,7 +72,7 @@ export const nativeAmount = (state: string = '0', action: Action) => {
     case 'UI/SEND_CONFIMATION/UPDATE_TRANSACTION': {
       if (!action.data) throw new Error('Invalid Action')
       if (!action.data.parsedUri) return state
-      return action.data.parsedUri.nativeAmount || state.nativeAmount || '0'
+      return action.data.parsedUri.nativeAmount || state || '0'
```

Comment:
This might actually create a bug, since `state.nativeAmount` would always be `undefined` (strings don't have this property). Just removing `state` and always defaulting to `'0'` would avoid changing functionality compared to what happens today.